rfcs/accessibility/accessibility.md

```diff
@@ -0,0 +1,76 @@
+
+## Problem statement
+
+Fluent UI components aim to fully respect both WCAG and ARIA and the Fluent UI team is working with the community to improve them. From a technical point of view, three main challenges were identified:
+
+1. Accessibility is a complex field that is often not taught and left behind in planning, design, development, and testing. It can be hard to achieve high quality user experiences and easy to regress them.
+2. Each screen reader and browser works differently, often making it hard or even impossible to achieve high quality user experience for all combinations. One of the biggest reasons behind this is that browsers and screen readers are often updated, with each update having the potential to change the user experience from one version to the next.
+3. Not all UI/UX designs can be sufficiently represented with the existing ARIA roles and patterns.
+
+## Detailed Design or Proposal
+
+To address these challenges, the following principles will be followed:
+
+- Challenge 1: Sufficient test coverage will be provided, with understandable and documented tests.
+- Challenge 2: Components will be created based on well tested prototypes or existing components. We will report and track issues with browsers and assistive tech vendors.
+- Challenge 3: We will work with the community to improve standards. We will maintain a set of best practices and validations that help consumers design accessible user interfaces.
+
+### Target accessibility areas
+
+- DOM structure that provides semantical value either by using correct element types or roles
+- ARIA attributes that are valid for elements/roles used and provide correct state information about the component or element
+- Keyboard navigation (navigation by tabbing, arrow keys, pagination or letter keys, click and right click (Enter/Space, Shift+F10), and close (Escape)) applied based on the component
+- Screen reader navigation (Virtual Cursor/Browse mode/Scan mode/VoiceOver keys)
+- Touch interaction
+- Focus handling when the component is able to move the focus in a predictable way - mostly when opening menus, popups or dialogs (autofocus) or dismissing them using Esc. Focus trap for Dialog and popups
+- Sufficient color contrast
+- Light, Dark and High Contrast themes
+- Displaying focus indicator when keyboard is used to interact with them
+- Tested against visual inconsistencies/bugs on a zoom up 400%
+
+Fluent UI components will use [ability-helpers](https://github.com/microsoft/ability-helpers) for focus handling functionality, so that they can be easily integrated with application-level ability-helpers functionality such as delooser and cross-iframe focusing.
```

Comment:
```suggestion
Fluent UI components will use [tabster](https://github.com/microsoft/tabster) for focus handling functionality, so that they can be easily integrated with application-level tabster functionality such as delooser and cross-iframe focusing.
```
Might just want to update for the sake of getting the new name out there

Reply:
not 100% sure if the 3rd instance of ability-helpers should be replaced with tabster, i'll leave it up to you